test/BUILD.gn

```diff
@@ -838,7 +838,10 @@ if (!is_android) {
     }
 
     if (enable_brave_vpn) {
-      deps += [ "//brave/components/brave_vpn" ]
+      deps += [
+        "//brave/components/brave_vpn",
+        "//brave/renderer/skus:browser_tests",
```

Comment:
this should not be under enable_brave_vpn

Reply:
done